src/api-sdks/connection-api/clients/python/ideastatica_connection_api/connection_api_client.py

```diff
@@ -7,28 +7,62 @@
 logger = logging.getLogger(__name__)
 
 class ConnectionApiClient:
-    def __init__(self, configuration: Configuration):
-        self.configuration = configuration
+    def __init__(self, base_url: str):
+        self.base_url = base_url
+        self.configuration = Configuration(host=self.base_url)
+        
+        self.client: Optional[api_client.ApiClient] = None
+        self.project_id: Optional[str] = None
```

Comment:
Can we please change this property name to active_project_id? in both Python and c#. The active_project_id should not be stored here. It should just be retrieved from the Project extension class. see this in c# https://github.com/idea-statica/ideastatica-public/blob/e2856fa1dcdf8dc63faa8dca8fcd9d676d34ff40/src/api-sdks/connection-api/clients/csharp/src/IdeaStatiCa.ConnectionApi/ConnectionApiClient.cs#L21

Reply:
remove from api client since it's stored in ProjectExt in ActiveProjectData